src/main/java/fr/lunatech/timekeeper/services/requests/OrganizationRequest.java

```diff
@@ -35,14 +35,14 @@ public OrganizationRequest(@NotBlank String name, @NotNull String tokenName) {
         this.tokenName = tokenName;
     }
 
-    public Organization unbind(@NotNull Organization organization, @NotNull AuthenticationContext ctx) {
+    public Organization unbind(@NotNull Organization organization) {
         organization.name = getName();
         organization.tokenName = getTokenName();
         return organization;
     }
 
     public Organization unbind(@NotNull AuthenticationContext ctx) {
```

Comment:
If you don't use the AuthenticationContext anymore, you may removed it from the method's parameter

Reply:
yes agree